pkgs/applications/misc/kratos/default.nix

```diff
@@ -1,8 +1,16 @@
-{ fetchFromGitHub, buildGoModule, lib, stdenv }:
-
-buildGoModule rec {
+{
+  fetchFromGitHub,
+  buildGoModule,
+  lib,
+  stdenv
+}:
+let
   pname = "kratos";
   version = "1.1.0";
+  commit = "f47675b82012e0ff74b05b9b7e713b3aa2fdda54";
```

Comment:
Updating the commit hash every time is not worth it. Might as well use a dummy string.

Reply:
Okay, I'll just remove it - it will say undefined as before